### ISSUE TITLE
Compilation fails if $script starts with a number

### ISSUE THREAD

**Issue by AlexHeylin** [JFLarvoire/SysToolsLib#5]
If you rename the script file to anything that starts with a number (like 321.ps1) then run` .\321.ps1 -setup `the compilation will fail and spit a few pages of errors like:
```
PS C:\Users\Alex\Desktop\> .\321.ps1 -setup
Add-Type : c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(43) : Identifier expected
c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(42) :
c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(43) : >>>   public class 321 : ServiceBase {
c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(44) :     private System.Diagnostics.EventLog eventLog;
      // EVENT LOG
At C:\Users\Alex\Desktop\321.ps1:861 char:5
+     Add-Type -TypeDefinition $source -Language CSharp -OutputAssembly ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (c:\Users\Alex\A...tifier expected:CompilerError) [Add-Type], Exception
    + FullyQualifiedErrorId : SOURCE_CODE_ERROR,Microsoft.PowerShell.Commands.AddTypeCommand

Add-Type : c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(43) : } expected
c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(42) :
c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(43) : >>>   public class 321 : ServiceBase {
c:\Users\Alex\AppData\Local\Temp\nbt05ch1.0.cs(44) :     private System.Diagnostics.EventLog eventLog;
      // EVENT LOG
At C:\Users\Alex\Desktop\321.ps1:861 char:5
+     Add-Type -TypeDefinition $source -Language CSharp -OutputAssembly ...
```
Sorry but I don't know C (of any variant) well enough to be able to suggest a fix.  
**Thanks VERY much for this template - it's really saved my backside!**

**Comment by AlexHeylin** [comment]
Forgot to say: PSService.ps1 is the script in question. Doh!

**Comment by JFLarvoire** [comment]
Service names beginning with a digit are unusual, but legal indeed Ex: 1394ohci
Bug fixed in commit c3a6d83, by prefixing the C# class name with the string "Service_".
This makes sure the class name begins with a letter in all cases, and thus prevents compilation failures.
